packages/core/helpers/mask.ts

```diff
@@ -0,0 +1,31 @@
+import VMasker from 'vanilla-masker'
+
+export const unMask = (value, mask) => {
+  if (mask === 'money') {
+    const valueNumerical = Number(value.replace(/\D/g, '')) / 100
+    return valueNumerical.toFixed(2)
+  }
+  return value.replace(/\W/g, '')
+}
+
+const masker = (value, pattern, options) =>
+  VMasker.toPattern(value, { pattern, ...options })
+
+const multimasker = (value, patterns, options) =>
+  masker(
+    value,
+    patterns.reduce(
+      (memo, pattern) => (value.length <= unMask(memo).length ? memo : pattern),
```

Comment:
I think that it always will be a diff. If i'm not wrong the value probably will come masked, `000.000.000-00` then we're comparing this with something without mask that can be '00000000000'. Another thing is, we're deciding which value will be returned based only on his length? and if i pass _ovo_ and the value on memo was _omo_ or _12345_ and _67890_ ? If with that being said and i'm still wrong can u add tests just this just to sanity check? cause if works as intended will be a very interesting case

Last but not least, whis whole file hasn't any type definition, so, what's the point to use TS?